### ISSUE TITLE
multiple job info modals

### ISSUE THREAD

**Issue by soxofaan** [Open-EO/openeo-web-editor#251]
(this probably is know or already reported, feel free to close)

In batch job overview, when I click the "I" button to see batch job details and if I'm fast enough  (faster than the back-end response, which is not that hard to beat :stuck_out_tongue: ) to click for multiple jobs , I can trigger multiple modals at the same time

screenshot:
![Screenshot from 2022-05-10 15-55-04](https://user-images.githubusercontent.com/44946/167646456-31c722a1-99ea-4d3d-93e1-b94e8b7d306a.png)

**Comment by m-mohr** [comment]
Yeah, I know. But opening multiple modals is totally fine, I think. ;-)

Maybe make the back-end faster? :-P
